app/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.tsx

```diff
@@ -288,10 +288,21 @@ export const DirectoryLiveTable: React.FC<DirectoryTableProps> = (props) => (
                 Looker
               </LinkExternal>
             )}
+            {experiment.monitoringDashboardUrl &&
+              experiment.resultsReady &&
+              experiment.rolloutMonitoringDashboardUrl && <br />}
+            {experiment.rolloutMonitoringDashboardUrl && (
+              <LinkExternal
+                href={experiment.rolloutMonitoringDashboardUrl!}
+                data-testid="link-rollout-monitoring-dashboard"
+              >
+                Rollout dashboard
+              </LinkExternal>
+            )}
             {experiment.monitoringDashboardUrl && experiment.resultsReady && (
               <br />
             )}
-            {experiment.resultsReady && (
+            {!experiment.isRollout && experiment.resultsReady && (
```

Comment:
We don't want to show results when the experiment is rollout?

Reply:
I don't think that rollouts have a Results page the way that experiments do (but now that you mention it... @jaredlockhart pls confirm)

Reply:
Yes that's correct, there is no 'results' for rollouts